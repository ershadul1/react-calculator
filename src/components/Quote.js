import React from 'react';

const Quote = () => (
  <div id="quote">
    <h1>Quote</h1>
    <p>
      “The difference between the poet and the mathematician is that
      the poet tries to get his head into the heavens while the mathematician
      tries to get the heavens into his head.”
      ― G.K. Chesterton
    </p>
  </div>
);

export default Quote;
