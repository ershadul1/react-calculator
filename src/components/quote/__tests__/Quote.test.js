import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../Quote';

it('Quote snapshot test', () => {
  const tree = renderer
    .create(<Quote />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
