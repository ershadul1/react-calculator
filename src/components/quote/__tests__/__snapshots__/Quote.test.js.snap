// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote snapshot test 1`] = `
<div
  id="quote"
>
  <h1>
    Quote
  </h1>
  <p>
    “The difference between the poet and the mathematician is that the poet tries to get his head into the heavens while the mathematician tries to get the heavens into his head.” ― G.K. Chesterton
  </p>
</div>
`;
